models/agent_model.go

```diff
@@ -172,21 +172,24 @@ func (s *Agent) DSN(service *Service, dialTimeout time.Duration, database string
 
 		return cfg.FormatDSN()
 
-	case MongoDBExporterType:
-		// TODO return MongoURI for the new driver
-		fallthrough
-
-	case QANMongoDBProfilerAgentType:
+	case QANMongoDBProfilerAgentType, MongoDBExporterType:
 		u := &url.URL{
 			Scheme: "mongodb",
 			Host:   net.JoinHostPort(host, strconv.Itoa(int(port))),
+			Path:   fmt.Sprintf("/%s", database),
 		}
 		switch {
 		case password != "":
 			u.User = url.UserPassword(username, password)
 		case username != "":
 			u.User = url.User(username)
 		}
+
+		if dialTimeout > 0 {
+			v := u.Query()
+			v.Set("connectTimeoutMS", strconv.Itoa(int(dialTimeout/time.Millisecond)))
+			u.RawQuery = v.Encode()
+		}
```

Comment:
There is a code below that does the same for PostgreSQL. Is there a reason why we do the same thing in two different ways?